### ISSUE TITLE
Issue with sst dev with NextJS

### ISSUE THREAD

**Issue by Marttyn** [sst/ion#1156]
Hi, I'm encountering an issue when trying to run sst dev in a Next.js project.

Here is my package.json:

```
"scripts": {
    "build": "next build",
    "dev": "sst dev next dev",
    "lint": "next lint",
  },
"dependencies": {
    "next": "14.2.4",
    "react": "^18",
    "sst": "ion",
     // other dependencies...
  },
```

Steps taken:

1. Ran `npm run dev`:

    - This starts both sst dev and the frontend, but live reload doesn't work. I have to rerun the command after every code change, which takes a long time.

2. Changed script to `"dev": "next dev"` and ran `sst dev`:

    - The frontend doesn't start automatically when I do this.

3. Ran `sst dev` and `next dev` in separate terminals:

    - This approach results in missing environment variables.

4. Ran `npx sst dev`:

    - This gives me the following error: `npm error could not determine executable to run.`

I'm running the commands in WSL (Ubuntu) while the project code is located on a Windows drive (using /mnt paths). I'm wondering if this might be causing the issue.

**Comment by jayair** [comment]
Which version of sst is this?

**Comment by Marttyn** [comment]
> Which version of sst is this?

That's something I found odd. In my package.json, the version was listed as "ion", which seemed incorrect. When I tried upgrading by running sst upgrade, it updated from version 0.0.473 (or something similar) to 3.1.46. However, I was still encountering errors when running sst dev and sst deploy.

To resolve the issue, I decided to move everything to WSL, reinstalled SST from scratch, and now it's working as expected. I'm not sure why it initially installed such an old version of SST, but after the reinstall, everything runs smoothly.

My guess why that the wrong version was installed and the cause for the errors was because the files were on Windows.
